saleor/static/scss/layouts/_product-list.scss

```diff
@@ -14,8 +14,13 @@
 .product-list {
   text-align: center;
   margin-bottom: $global-margin * 2;
-  img {
-    max-width: 100%;
+  .img-wrapper {
```

Comment:
I did not test the changes yet, but could you refactor a bit to avoid repeating the code?

Reply:
> I did not test the changes yet, but could you refactor a bit to avoid repeating the code?

Any thoughts on where to put the duplicated code? Actually they belong to two different layout indeed. And they don't belong to any components obviously.

Reply:
I think, we could safely put it above `.product-list` in this current file (`layouts/_product-list.scss`). Maybe rename `.img-wrapper` to `.product-img-wrapper` or something else to be more explicit?

Reply:
Thanks for your tips. I've already refactored it according to your first suggestion.